app/lib/pre_assembly/digital_object.rb

```diff
@@ -53,14 +52,14 @@ def setup
       self.technical_md_file  = 'technicalMetadata.xml'
       self.content_md_xml     = ''
       self.technical_md_xml   = ''
-      self.content_structure  = (project_style ? project_style[:content_structure] : 'file')
+      self.content_structure  = (project_style ? project_style : 'file')
     end
 
     def stager(source, destination)
-      if staging_style.nil? || staging_style == 'copy'
-        FileUtils.cp_r source, destination
-      else
+      if staging_style
```

Comment:
shouldn't this be staging_style_symlink?   Possibly fixed in later commit?  (Do we have tests for the two styles?  If not, please ticket.  If we're offering an option, we should definitely have tests for the different values.

Reply:
yup, i had the same question

Reply:
ah, i think i see... looks like `staging_style` on the `DigitalObject` instance is set to `staging_style_symlink` in `Bundle#digital_object_base_params`?

Reply:
No: on the "inside" of `DigitalObject`, it is still `staging_style` (now a boolean).  This is a shortfall of a "Hashes in, Hashes out" approach to class design.  This is most visible from `Bundle#digital_object_base_params` where you can see the keys coming in.  

I don't think pursuit of consistency between `Bundle` and DO is high value, relative to making DigitalObject initialize **with** a bundle object (thereby converting all the INIT_PARAMS to things delegated to the bundle and making OO actually work).

Reply:
might be nice to make the names line up, possibly in a follow on PR instead of this one.

Reply:
Again, I disagree that that is valuable relative to actually using OO, which would make the divergence of names impossible.

Reply:
and I'm okay with it being ticketed separately.

Reply:
ticket created #271

Reply:
If you keep the names the same you *still* have similar-but-different, just between the YAML (hash) structure that is still hanging around and the model's booelan value.